packages/react/src/Button/Button.tsx

```diff
@@ -45,44 +47,47 @@ export const Button = forwardRef(
     const [isFocused, setIsFocused] = React.useState(false)
     const Component = as || 'button'
 
+    const isDisabled =
```

Comment:
Added a new variable `isDisabled`. Since `aria-disabled` can take a value of `string` or `boolean` (e.g. `aria-disabled="true"` === `aria-disabled={true}`) I made a conditional to verify that it either has a string value of `"true"` or a boolean value of `true`.